fast64_internal/sm64/sm64_collision.py

```diff
@@ -608,41 +604,17 @@ class SM64_ExportCollisionPanel(SM64_Panel):
     bl_label = "SM64 Collision Exporter"
     goal = "Object/Actor/Anim"
 
+    @classmethod
+    def poll(cls, context):
+        return context.scene.fast64.sm64.export_type != "C"
```

Comment:
just overriding the poll doesn´t work and will cause sm64 panels to show in oot, tested this just now by changing to binary and then switching game modes